address/list.js

```diff
@@ -0,0 +1,96 @@
+var addr_default = [5,6,7,8];
+
+var loc = window.location.pathname;
+var split_url = loc.split('/');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8393720)
```suggestion
var split_url = loc.split("/");
```